src/pages/ResearcherConsole.js

```diff
@@ -135,7 +135,7 @@ class ResearcherConsole extends Component {
       }
     );
 
-    DAR.getPartialDarRequestList().then(
+    DAR.getDraftDarRequestList().then(
```

Comment:
New method is working well, the data is populating on the table and is redirecting as it should